### ISSUE TITLE
Method find doesn't raise errors if Mongo server is down

### ISSUE THREAD

**Issue by golfarelli** [balderdashy/sails-mongo#231]
Hi, i'm new to sails-mongo.
I'm using waterline in express app with sails-mongo adapter.
I have create this action:
```javascript
app.get('/users', function(req, res) {
  app.models.user.find().exec(function(err, models) {
    if(err) return res.json({ err: err }, 500);
    res.json(models);
  });
});
```
During my tests i have stopped mongo server and launch the action "/users", but i don't received any errors and the request remain in append.

Do i set some specific configuration for connection timeout to mongo server ?

**Comment by sailsbot** [comment]
Thanks for posting, @golfarelli.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
